### ISSUE TITLE
Automatch Doesn't Show Options Dialogue

### ISSUE THREAD

**Issue by drheld** [aiannacc/Goko-Salvager#252]
Steps to reproduce:
1) Click the "Automatch: Idle" button.

Expected results:
An options dialogue to select automatch criteria is shown as per:
https://github.com/aiannacc/Goko-Salvager/wiki/Automatch

Actual results:
Changes to "Automatch: Searching".

**Comment by aiannacc** [comment]
Updated the wiki page to match the current version's behavior.

Was your concern only the mismatch, or do you feel that hiding the automatch options in the User Settings dialog is confusing?

**Comment by drheld** [comment]
I would never have found it there -- but I didn't know to look for the User Settings. Updating the help let me figure it out.

It might be nice to have a gear or "..." button to allow changing it from the lobby, but up to you.

**Comment by aiannacc** [comment]
I agree.  And maybe an explanatory message the first time you click it.  I'll leave this issue open.
